google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -28,7 +28,7 @@
 
   private static final String BIGQUERY_SCOPE = "https://www.googleapis.com/auth/bigquery";
   private static final Set<String> SCOPES = ImmutableSet.of(BIGQUERY_SCOPE);
-  private static final long serialVersionUID = -8592198255032667206L;
+  private static final long serialVersionUID = -2437598817433266049L;
```

Comment:
See [Type Changes Affecting Serialization](http://docs.oracle.com/javase/7/docs/platform/serialization/spec/version.html#6678) for some info on this.